requirements/edx/testing.txt

```diff
@@ -117,14 +117,14 @@ edx-proctoring-proctortrack==1.0.5  # via -r requirements/edx/base.txt
 edx-proctoring==2.4.4     # via -r requirements/edx/base.txt, edx-proctoring-proctortrack
 edx-rbac==1.3.2           # via -r requirements/edx/base.txt, edx-enterprise
 edx-rest-api-client==5.2.1  # via -r requirements/edx/base.txt, edx-enterprise, edx-proctoring
-edx-search==1.4.1         # via -r requirements/edx/base.txt
+git+https://github.com/raccoongang/edx-search.git@golub-sergey/BD-19/transition-from-ES1.5-to-ES7  # via -r requirements/edx/base.txt
```

Comment:
And here.

Reply:
> We got conflicts here. Please resolve them.

this requirement needs for testing edx-search with edx-platform changes. Sorry for disinformation.
I've made pr to the edx-platform without requirements update. edx#24915
I suppose that edx-platfrom and edx-search changes are applied together